src/services/news.ts

```diff
@@ -55,11 +61,12 @@ export async function setNewsRead <T extends Account> (account: T, message: Info
     case AccountService.Local: {
       break;
     }
+    case AccountService.EcoleDirecte: { break; }
     case AccountService.UPHF: {
       break;
     }
```

Comment:
nit: juste pour que ce soit plus simple à lire, rien de bien important

```suggestion
    case AccountService.Local:
    case AccountService.EcoleDirecte:
    case AccountService.UPHF:
      break;
```